lib/formats/base64.js

```diff
@@ -1,15 +1,16 @@
+class Base64Parser {
+  constructor(decorator) {
+    this.decorator = decorator;
+  }
 
-function Parser(decorator) {
-  this.decorator = decorator;
-}
+  parse(payload) {
+    let payloadToParse = payload;
+    if (this.decorator) {
+      payloadToParse = this.decorator.parse(payload);
```

Comment:
This LGTM but I don’t think the intermediary `payloadToParse` variable is really necessary is it? It would work just fine as

```js
payload =  this.decorator.parse(payload);
```

Reply:
Oh, I wasn't changing any of the code functionality in the PR. Here I just renamed the variable.

Reply:
I'm just going to focus this PR and other PRs on converting this file to es6 and not changing the functionality. I'd rather do that in a different PR.